pulumi/grapl/__main__.py

```diff
@@ -479,6 +502,9 @@ def main() -> None:
         vars=grapl_core_vars,
         opts=pulumi.ResourceOptions(
             provider=nomad_provider,
+            depends_on=[
+                nomad_graph_db.job,
+            ],
```

Comment:
we _could_ try deploying them at the same time; i actually thought this was a decent way to get some "ordering" here (you wouldn't want our other services starting up before our graph dbs were ready, right?)